update base instructions (#836)
axelarnetwork/axelar-docs: src/pages/validator/external-chains/base.mdx

git clone https://github.com/base-org/node
```

2. In `docker-compose.yml`, uncomment the right .env file according to the network you want to run it on

```bash
    env_file:
      # select your network here:
#      - .env.sepolia
#      - .env.mainnet

> ```

Reviewer: This impacts Optimism as well (and potentially Mantle if you can take a look)

Author: Yes, will update other l2 too in a different PR

Update the following variables in your `.env` file
```bash

> # [recommended] replace with your preferred L1 (Ethereum, not Base) node RPC URL:
> OP_NODE_L1_ETH_RPC=https://rpc.sepolia.org

Reviewer: ```suggestion
# Set the RPC url for your own L1 (Ethereum) node:
OP_NODE_L1_ETH_RPC=[your ethereum RPC]
```

> # [required] replace with your preferred L1 CL beacon endpoint:
> OP_NODE_L1_BEACON=https://your.sepolia.beacon.node/endpoint-here

Reviewer: ```suggestion
# Set the url for your own L1 CL beacon endpoint:
OP_NODE_L1_BEACON=[your beacon node]
```

> OP_NODE_L1_BEACON=https://your.sepolia.beacon.node/endpoint-here

Reviewer: Does the Ethereum setup indicate how to get the beacon endpoint? If not, can you add some instructions to it, and link to that here?

Author: I think docs should have it already, will add a link here

Author: Added
